src/gui/wizard/owncloudwizardcommon.cpp

```diff
@@ -130,18 +130,34 @@ namespace WizardCommon {
 
     void customizeSecondaryButtonStyle(QAbstractButton *button)
     {
-        auto pushButtonPalette = button->palette();
-        pushButtonPalette.setColor(QPalette::Button, Theme::instance()->applicationBackgroundColor());
-        pushButtonPalette.setColor(QPalette::ButtonText, Theme::instance()->applicationTitleColor());
-        button->setPalette(pushButtonPalette);
+        const auto theme = Theme::instance();
+
+        // FIXME: For whatever reason I got crashes if i concat the string directly with +
```

Comment:
I suspect it's because you received it in an auto variable, shouldn't crash if you receive it in a QString variable. This is because + doesn't emits QString type but QStringBuilder type which is coerced into QString only later when you call setStyleSheet. But QStringBuilder is lazy evaluated and the temporaries will be gone at that point => boom.

That's why by receiving in a QString you coerce the type at the right time and it won't crash. This is one of the downsides of auto with QString. I fall for it regularly.

Reply:
Interesting :) Thanks